### PR TITLE
Final

### PR DESCRIPTION
**Pull request by MillsProvosty**
## Changes proposed in this pull request:

Resolves #

## The following checks have been completed:

 - [ ] Tested new feature(s) as well as any feasible edge cases
 - [ ] Ran the test suite - all tests are passing
 - [ ] Checked affected endpoints in Postman

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,16 +14,39 @@ Learning goals were as follows:
...
+Install the following dependencies for testing:
 
-## Running Tests
 - install Jest with ``` npm install jest -g```
 - install packages ``` npm install babel-jest supertest shelljs -D ```
 
+To securely store API keys:
+- Install dotenev with ```npm install dotenv``` and put ```require('dotenv').config()``` in the app.js file
+- In the .env file: ```DARK_SKY_API_KEY=<your api key here>``` and  ```GOOGLE_GEOCODE_API_KEY=<your api key here>```
+- Add .env to the .ignore file.
```
Comment (smainar) [inline README.md]: `.ignore` should be `.gitignore`

File: README.md [hunk 2]
```diff
@@ -14,16 +14,39 @@ Learning goals were as follows:
...
+
+Install the following dependencies for testing:
 
-## Running Tests
 - install Jest with ``` npm install jest -g```
 - install packages ``` npm install babel-jest supertest shelljs -D ```
 
+To securely store API keys:
+- Install dotenev with ```npm install dotenv``` and put ```require('dotenv').config()``` in the app.js file
+- In the .env file: ```DARK_SKY_API_KEY=<your api key here>``` and  ```GOOGLE_GEOCODE_API_KEY=<your api key here>```
```
Comment (smainar) [inline README.md]: After line 30, my suggestion is to add the following:

Create a .env file in the root directory of your project.

Comment (MillsProvosty) [reply README.md]: Excellent, thank you

**Comment by smainar** [comment]
@MillsProvosty - I made a couple comments regarding your README.md. 

Also, I see that you installed the `request` package in order to make http calls. I'm unfamiliar with how it works because I used `node-fetch`. If you are interested in looking into `node-fetch`, below is the documentation I used for it:

https://www.npmjs.com/package/node-fetch.)
